### ISSUE TITLE
Can I download tracks without vulcan support?

### ISSUE THREAD

**Issue by DimasterEZ** [flyingrub/scdl#365]
I tried to download music but there was an error appeared in the form of missing vulcan-1.dll. My GPU is too old... Any ideas?

**Comment by flyingrub** [comment]
Can you show me the full log ? That's odd that it require vulkan.

**Comment by DimasterEZ** [comment]
> Can you show me the full log ? That's odd that it require vulkan.
Well actually ffmpeg required this file.

**Comment by gpchelkin** [comment]
@DimasterEZ which FFmpeg version do you have? Try to use latest build without vulkan:
https://github.com/BtbN/FFmpeg-Builds/releases

**Comment by DimasterEZ** [comment]
@gpchelkin yes I didn't notice that I downloaded the version with vulkan support. Now it works. Thanks!
